CHANGELOG.md

```diff
@@ -53,7 +53,7 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
-* Fix runpath_search_paths for inherit !search_path targets  
+* Fix runpath_search_paths for `inherit !search_path` targets  
```

Comment:
nit of the nits you can ignore me but I think its `inherit!` not `inherit !`

Reply:
Thanks! Fixed and squashed.